src/main/java/tds/content/repositories/impl/S3ItemDataRepository.java

```diff
@@ -72,9 +73,11 @@ public InputStream findResource(final String resourcePath) throws IOException {
             return item.getObjectContent();
         } catch (final AmazonS3Exception ex) {
             if (ex.getStatusCode() == HttpStatus.SC_NOT_FOUND) {
-                throw new NotFoundException(String.format("Could not find resource at resource location %s", resourcePath));
+                throw new NotFoundException(String.format("Could not find resource at resource location %s", resourceLocation));
```

Comment:
What do you think about adding some logging in here with the actual s3 Exception?